### PR TITLE
Build endpoint /features/brief and write feature brief to DB

### PR DESCRIPTION
**Pull request by MahtabBukhari**
## Evidence:

https://www.loom.com/share/fcd4b3d5a4994ceeb38532453bda6b78?sid=96bec932-bfa2-4ade-a93b-b1786a618948

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by MahtabBukhari** [comment]
@humansinstitute Could you please review PR as tests are pass?

**Comment by humansinstitute** [comment]
@MahtabBukhari can you address the failing test? 

```
  Running:  53_workspaceMission.cy.ts                                                     (45 of 59)


  Create Workspace And Update Mission
    1) Creating A Workspace


  0 passing (15s)
  1 failing

  1) Create Workspace And Update Mission
       Creating A Workspace:
     AssertionError: Timed out retrying after 4000ms: Expected to find content: 'carol' but never did.
      at Context.eval (webpack://catcher/./cypress/support/commands.ts:385:0)




  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        1                                                                                │
  │ Passing:      0                                                                                │
  │ Failing:      1                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  1                                                                                │
  │ Video:        false                                                                            │
  │ Duration:     15 seconds                                                                       │
  │ Spec Ran:     53_workspaceMission.cy.ts                                                        │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘


  (Screenshots)

  -  /home/runner/work/sphinx-tribes/sphinx-tribes/tribes-frontend/cypress/screenshot     (1280x720)
     s/53_workspaceMission.cy.ts/Create Workspace And Update Mission -- Creating A Wo               
     rkspace (failed).png           
```

**Comment by MahtabBukhari** [comment]
@humansinstitute let me check

**Comment by MahtabBukhari** [comment]
@humansinstitute Could you please review PR now?

**Review by tobi-bams — CHANGES_REQUESTED** [review]

File: handlers/features.go [hunk 1]
```diff
@@ -164,6 +172,71 @@ func (oh *featureHandler) GetFeatureByUuid(w http.ResponseWriter, r *http.Reques
 	json.NewEncoder(w).Encode(workspaceFeature)
 }
 
+func (oh *featureHandler) UpdateFeatureBrief(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
```
[the remark this answers is not in the document]
Comment (MahtabBukhari) [reply handlers/features.go]: you can review again as checks pass

**Review by tobi-bams — APPROVED** [review]

**Pull request merged**